code/tick/feed.q

```diff
@@ -31,15 +31,37 @@ randomize:{value "\\S ",string "i"$0.8*.z.p%1000000000}
 rnd:{0.01*floor 0.5+x*100}
 vol:{10+`int$x?90}
 
-/ randomize[]
-\S 235721
+randomize[]
+/ \S 235721
 
-/ =========================================================
+/ =========================================================================================
+/ generate weights to stop even distribution of counts and sizes
+
+weight:0.3 0.6 1.8 1.1 1.2 1.6 0.9 0.4 0.7 1.4
+
+/ assign multipliers to skew size columns
+volmap:s!-10?weight
```

Comment:
think `neg[cnt]` should be used here instead of hardcoding `-10`, so it's still easy to add/remove syms?